recipes/agent.rb

```diff
@@ -51,6 +53,13 @@
   chef_environment: node.chef_environment
 }
 
+# set lumberjack key locations and perms
+node.default['lumberjack']['ssl_dir'] = node['logstash']['instance_default']['basedir']
+node.default['lumberjack']['ssl_key_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl key']}"
+node.default['lumberjack']['ssl_cert_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl certificate']}"
+node.default['lumberjack']['user'] = node['logstash']['instance_default']['user']
+node.default['lumberjack']['group'] = node['logstash']['instance_default']['group']
+
```

Comment:
The problem there is that these are meant to replace what is there by default.  By design node['lumberjack'] attributes are set with defaults, then the agent and forwarder recipies override them with their own seperate settings.   If someone want's to override completely they will need to use node.override.